lib/widgets/content.dart

```diff
@@ -896,12 +911,18 @@ class _KatexSpan extends StatelessWidget {
     }
 
     final styles = node.styles;
+    // We expect vertical-align to be only present with the
+    // `strut` span, for which parser explicitly emits `KatexStrutNode`.
+    // So, this should always be null for non `strut` spans.
+    assert(styles.verticalAlignEm == null);
 
     final fontFamily = styles.fontFamily;
     final fontSize = switch (styles.fontSizeEm) {
       double fontSizeEm => fontSizeEm * em,
       null => null,
     };
+    if (fontSize != null) em = fontSize;
```

Comment:
Cool, this is a good idea for making sure we don't forget `fontSize ??` in any `em` references below.